src/js/views/maps/ViewFinderView.js

```diff
@@ -0,0 +1,155 @@
+"use strict";
+
+define([
+  "backbone",
+  "text!templates/maps/view-finder.html",
+], (
+  Backbone,
+  Template,
+) => {
+  /**
+   * @class ViewFinderView
+   * @classdesc The ViewFinderView allows a user to search for a latitude and longitude in the map view.
+   * @classcategory Views/Maps
+   * @name ViewFinderView
+   * @extends Backbone.View
+   * @screenshot views/maps/ViewFinderView.png
+   * @since 2.27.1
+   * @constructs ViewFinderView
+   */
+  var ViewFinderView = Backbone.View.extend({
+    /**
+     * The type of View this is
+     * @type {string}
+     */
+    type: "ViewFinderView",
+
+    /**
+     * The HTML classes to use for this view's element
+     * @type {string}
+     */
+    className: classNames.baseClass,
+
+    /**
+    * The events this view will listen to and the associated function to call.
+    * @type {Object}
+    */
+    events: {
+      [`click .${classNames.button}`]: 'search',
+      [`keyup .${classNames.input}`]: 'keyup',
+    },
+
+    /** 
+     * Values meant to be used by the rendered HTML template.
+     */
+    templateVars: {
+      errorMessage: "",
+      inputValue: "",
+      placeholder: "Search by latitude and longitude",
+      classNames,
+    },
+
+    /**
+     * @typedef {Object} ViewFinderViewOptions
+     * @property {Map} The Map model associated with this view allowing control
+     * of panning to different locations on the map. 
+     */
+    initialize(options) {
+      this.mapModel = options.model;
+    },
+
+    render() {
+      this.focusInput();
```

Comment:
Not familiar with the backbone life cycles yet- are we calling focusInput both here and in action() because render is only called once, but action is called every time user navigates to that tab?

Reply:
`ToolbarView` actually renders each of it's sections' corresponding views immediately, and then shows and hides them. Unfortunately that means that when the input becomes visible, the `render` function has already been called. The `render` function is called manually and it seems that most views call it in their `initialize` function which is similar to a `constructor` function.

Reply:
nit: should we remove the focusInput() here if it's gonna be called again in action() anyway?

Reply:
Action is *only* called when the user clicks the magnifying glass button to make this "View finder" panel visible; so only called once. After the user types "enter" or clicks the search button (on the input) I want the input to stay focused, so we need to call focus after each render.